tests/fraction_tests.ts

```diff
@@ -3,18 +3,30 @@
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
 
-VF.Test.Fraction = (function () {
-  var Fraction = {
+import { Fraction } from '../src/fraction';
+
+declare const QUnit: any;
```

Comment:
Can we move these declarations to a common file instead of repeating it everywhere?

Reply:
yes, like `tests_declarations.d.ts` which we can import for every test

Reply:
It is a good idea, I will also push the minor change on music_tests (avoidance of var).

Reply:
Sounds good. Let's just call it `declarations.d.ts` since `tests` is implied by the directory name.

Reply:
Sorry but I do not figure out how the content of declarations.d.ts should be so as not to get an error. Can you help?

Reply:
here: https://github.com/0xfe/vexflow/pull/878

Reply:
Changes done